doc/source/user_guide/enhancingperf.rst

```diff
@@ -562,6 +562,59 @@ Now let's do the same thing but with comparisons:
    of type ``bool`` or ``np.bool_``. Again, you should perform these kinds of
    operations in plain Python.
 
+The ``Series.eval`` method
+
+A new function added alongside the ``Series.query`` method that allows
+you to evaluate an expression in the "context" of a :class:`~pandas.Series`.
+
+.. ipython:: python
+   
+   index = list(range(2010, 2015))
+   data = np.random.randn(5)
+   series = pd.Series(data, index=index, name='data')
+   series.index.name = 'years'
+   series
+
+   series.eval('(years > 2010) & (data > 0)')
```

Comment:
Maybe change `(data > 0)` to `(years != 2011)` since we can't evaluate the data itself unfortunately